actioncable/lib/action_cable/route_wrapper.rb

```diff
@@ -0,0 +1,23 @@
+module ActionCable
+  module RouteWrapper
+
+    def internal_cable_path?
+      path =~ %r{\A#{self.class.action_cable_url}\z}
+    end
+
+    def internal?
+      super || internal_cable_path?
+    end
+
+    def self.included(klass)
+      klass.class_eval do
+        def internal_with_cable?
+          internal_without_cable? || internal_cable_path?
+        end
+
+        alias_method :internal_without_cable?, :internal?
+        alias_method :internal?, :internal_with_cable?
+      end
+    end
```

Comment:
This module is prepended, so I don't see why we need this whole `class_eval` dance. Calling `super` from `internal?` aught to be enough.